data/hypervisor.args.in

```diff
@@ -12,7 +12,7 @@ memory-backend-file,id=mem0,mem-path=@IMAGE@,size=@SIZE@
 -kernel
 @KERNEL@
 -append
-@KERNEL_PARAMS@
+@KERNEL_PARAMS@ @KERNEL_NET_PARAMS@
```

Comment:
That is because it is actually a concatenation to the same param (i.e kernel command line). Hence we have to put it in the same line.